libraries/AP_InertialSensor/AP_InertialSensor_Invensense.cpp

```diff
@@ -434,6 +449,31 @@ bool AP_InertialSensor_Invensense::update() /* front end */
 
     _publish_temperature(_accel_instance, _temp_filtered);
 
+    if (fast_reset_count) {
+        // check if we have reported in the last 1 seconds or
+        // fast_reset_count changed
+#if HAL_GCS_ENABLED
```

Comment:
Do we really need this level of debug? It's 300+ bytes on all boards and this seems like a significant contribution